source/administration/config-settings.rst

```diff
@@ -1089,10 +1089,12 @@ The public authentication certificate issued by your Identity Provider.
 
 Verify Signature
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-``true``: When true, Mattermost verifies that the signature sent from the SAML Response matches the Service Provider Login URL.
+**True**: Mattermost verifies that the signature sent from the SAML Response matches the Service Provider Login URL.
+
+**False**: Verification is disabled. Not recommended for production environments. For testing only.
```

Comment:
I don't think so, but if the description starts out with "Not recommended..." then it's more clear that trouble awaits those who are not true.